### DISCUSSION TITLE
PROPOSAL: Decrease minimum deposit for proposals

### DISCUSSION THREAD

**Discussion by zanglang** [crypto-org-chain/chain-main#648] [Ideas] [answered]
Currently the minimum deposit amount required for raising a governance proposal is 20,000 CRO, which amounts to about US$3k at time of writing, or $5k if using CRO's all-time-high price.

To improve decentralization and encourage community discussion, we could bring down this value to be more in-line with other Cosmos ecosystem projects. For example:

Akash = 1000 AKT ~ $3.6k
Osmosis = 500 OSMO ~ $1.9k
Cosmos = 64 ATOM ~ $1.5k
Secret = 100 SCRT ~ $161
Iris = 1000 IRIS ~ $110

Considering that the Crypto.org chain is still in its early stages, I would propose setting this value to 5,000 ~ 10,000 CRO. There should be no drawbacks for setting it too low, as CDC's voting power alongside its chain partners should be sufficient to safeguard from malicious proposals.

**Comment by eeldivad** [comment]
I agree.  Surprised how few proposals are submitted, but then I remembered the high deposit amount and it makes sense.  I think there was only 1 proposal submitted by non-CDC community member and it got rejected.  Does anyone know if he got his deposit back for the timeout proposal that was rejected?

**Comment by zanglang** [comment]
Bumping this again with updated prices.

With the recent bull-rush, currently the minimum deposit amount required for raising a governance proposal is 20,000 CRO, which amounts to almost US$14k at time of writing, or $19k if using CRO's all-time-high price.

To improve decentralization and encourage community discussion, we NEED to bring down this value to be more in-line with other Cosmos ecosystem projects. For example:

Akash = 1000 AKT ~ $2k
Osmosis = 500 OSMO ~ $5.3k
Cosmos = 64 ATOM ~ $1.6k
Secret = 100 SCRT ~ $585
Iris = 1000 IRIS ~ $117

Considering that the Crypto.org chain is still in its early stages, I would propose setting this value to 3,000 ~ 5,000 CRO (approximately US$2-3k)... which was where we started. There should be no drawbacks for setting it too low, as CDC's voting power alongside its chain partners should be sufficient to safeguard from malicious proposals.

**Comment by lezzokafka** [comment, marked as answer]
Hello @zanglang,
Thanks for the proposal and the references provided! 

The minimum deposit is indeed an anti-spamming mechanism while It should not be a barrier that stops shareholders from raising a governance proposal at the same time.

Judging by the numbers and analysis provided, I suggest we could reduce it by 50% to `10,000 CRO` as a first step and review it afterwards. Please let me know your thoughts on this. 👍

**Reply by zanglang** [reply]
Sounds good.

Kindly let us know when internal testing and/or testing on Croeseid has completed and we may proceed with putting up the proposal on mainnet, thanks!

**Reply by Terra854** [reply]
Why not we reduce it further to 5000 CRO (25% of current deposit)? It will allow more smaller stakeholders to put up proposals without making it too cheap.
